frontend/src/components/common/header/header.tsx

```diff
@@ -40,28 +58,44 @@ const Header: FC = () => {
           </Link>
         </div>
         {isRoot && <SearchBar onSearch={handleSearch} />}
-        <div className={styles.userWrapper}>
-          {hasUser ? (
-            <button onClick={handlePopupOpen} className={styles.button}>
-              <div className={styles.imageWrapper}>
-                <Image
-                  width="50"
-                  height="50"
-                  src={defaultUserAvatar}
-                  alt="user avatar"
-                  isCircular
-                  classes={styles.img}
-                />
-              </div>
-            </button>
-          ) : (
-            <div className={styles.buttonsWrapper}>
-              <Button label="Sign In" btnColor="gray" to={AppRoute.SIGN_IN} />
-              <Button label="Sign Up" btnColor="blue" to={AppRoute.SIGN_UP} />
+
+        <div className={styles.rightWrapper}>
+          {hasUser && (
+            <div className={styles.chatLogoWrapper}>
+              <Link to={AppRoute.CHATS}>
+                <div className={styles.chatImage}>
+                  {hasUnreadMessages ? (
+                    <Icon name="chatUnread" className={styles.chatIcon}></Icon>
+                  ) : (
+                    <Icon name="chat" className={styles.chatIcon}></Icon>
+                  )}
```

Comment:
If the only difference is a name
```suggestion
                    <Icon name={hasUnreadMessages ? 'chatUnread' : 'chat'} className={styles.chatIcon} />
```